test/e2e/framework/pod/utils.go

```diff
@@ -231,3 +231,13 @@ func mixinRestrictedContainerSecurityContext(container *v1.Container) {
 		}
 	}
 }
+
+// FindPodConditionByType loops the pod condition by type in the pod status
```

Comment:
nit godoc: `FindPodConditionByType loops through all pod conditions in pod status and returns the specified condition.`

Reply:
updated